main.go

```diff
@@ -34,7 +34,7 @@ func transferFunds(ctx context.Context, tx pgx.Tx, from int, to int, amount int)
 }
 
 func main() {
-	config, err := pgx.ParseConfig("postgresql://maxroach@localhost:55344/bank?sslmode=disable")
+	config, err := pgx.ParseConfig("postgres://{username}:{password}@{hostname}:{port}?sslmode=require")
```

Comment:
```suggestion
	config, err := pgx.ParseConfig("postgres://{username}:{password}@{hostname}:{port}/bank?sslmode=require")
```

Otherwise, it will attempt to create the tables in `defaultdb`, and the user doesn't have perms on `defaultdb`.

Reply:
Done.